src/containers/EditListingPage/EditListingWizard/EditListingDetailsPanel/EditListingDetailsPanel.js

```diff
@@ -271,6 +314,16 @@ const EditListingDetailsPanel = props => {
               ...rest
             } = values;
 
+            // Category levels are dynamic, i.e. in theory you can always nest
+            // another subcategory under a subcategory, therefore  we loop through them
+            const categories = {};
```

Comment:
```suggestion
            let categories = {};
```

Since you are going to mutate the content of this object, mark it explicitly with the usage of 'let' instead of 'const'.